frontend/src/app/person/person.component.html

```diff
@@ -1,50 +1,79 @@
     </div>
     <div class="form-group row">
-      <label class="col-sm-6 col-form-label">Est adhérent</label>
+      <label class="col-sm-6">Est adhérent</label>
       <div class="col-sm-6">
         <p id="adherent" class="form-control-static">{{ person.adherent ? 'Oui' : 'Non' }}</p>
       </div>
     </div>
     <div class="form-group row">
-      <label class="col-sm-6 col-form-label">Genre</label>
+      <label class="col-sm-6">Genre</label>
       <div class="col-sm-6">
         <p id="gender" class="form-control-static">{{ person.gender | displayGender }}</p>
       </div>
     </div>
     <div class="form-group row">
-      <label class="col-sm-6 col-form-label">Né(e) le</label>
+      <label class="col-sm-6">Né(e) le</label>
       <div class="col-sm-6">
         <p id="birthDate" class="form-control-static">{{ person.birthDate | date }}</p>
       </div>
     </div>
+    <div class="form-group row">
+      <label class="col-sm-6">Nom de naissance</label>
+      <div class="col-sm-6">
+        <p id="birthName" class="form-control-static">{{ person.birthName }}</p>
+      </div>
+    </div>
     <div class="form-group row" *ngIf="person.mediationEnabled">
-      <label class="col-sm-6 col-form-label">Statut marital</label>
+      <label class="col-sm-6">Statut marital</label>
       <div class="col-sm-6">
         <p id="maritalStatus" class="form-control-static">{{ person.maritalStatus | displayMaritalStatus }}</p>
       </div>
     </div>
     <div class="form-group row" *ngIf="person.mediationEnabled">
-      <label class="col-sm-6 col-form-label">Couverture médicale</label>
+      <label class="col-sm-6">Numéro de Sécurité Sociale</label>
+      <div class="col-sm-6">
+        <p id="socialSecurityNumber" class="form-control-static">{{ person.socialSecurityNumber }}</p>
+      </div>
+    </div>
+    <div class="form-group row" *ngIf="person.mediationEnabled">
+      <label class="col-sm-6">Numéro CAF</label>
+      <div class="col-sm-6">
+        <p id="cafNumber" class="form-control-static">{{ person.cafNumber }}</p>
+      </div>
+    </div>
+    <div class="form-group row" *ngIf="person.mediationEnabled">
+      <label class="col-sm-6">Couverture médicale</label>
       <div class="col-sm-6">
         <p id="healthCareCoverage" class="form-control-static">{{ person.healthCareCoverage | displayHealthCareCoverage }}</p>
       </div>
     </div>
+
+    <hr class="separator">
+
+    <h2>Informations</h2>
+
     <div class="form-group row" *ngIf="person.mediationEnabled">
-      <label class="col-sm-6 col-form-label">Entré(e) en France le</label>
+      <label class="col-sm-6">Entré(e) en France le</label>
       <div class="col-sm-6">
         <p id="entryDate" class="form-control-static">{{ person.entryDate | date }}</p>
       </div>
     </div>
+    <div class="form-group row" *ngIf="person.mediationEnabled">
+      <div class="col-sm-6"><span class="fa fa-calendar"></span> Date 1<sup>er</sup> RDV médiation</div>
```

Comment:
I'd remove the icon. It's the only field which has both a label and an icon, and I don't see why this field has a special treatment.

Reply:
:+1: